mycqu/card.py

```diff
@@ -190,46 +176,44 @@ class Card:
     """
     校园卡及其账单信息
     """
+    card_id: int
+    """校园卡id"""
     amount: float
     """账户余额"""
-    bills: List[Bill]
-    """最近交易记录"""
 
     @staticmethod
-    def fetch(session: Session, duration: int = 30) -> Card:
-        res = get_bill_raw(session, duration)
-        bills = []
-        for bill in res['bills']:
-            bills.append(Bill.from_dict(bill))
+    def fetch(session: Session) -> Card:
+        """
+        从card.cqu.edu.cn获取校园卡信息
+
+        :param session: 登录了统一身份认证（:func:`.auth.login`）并在 card.cqu.edu.cn 进行了认证（:func:`.mycqu.card_login`）的 requests 会话
+        :type session: Session
+        :raises CQUWebsiteError: 当网页获取状态码不为0000时抛出
+        :return: 获取的校园卡信息
+        :rtype: Card
+        """
+        card_info = get_card_raw(session)
 
         return Card(
-            amount=float(res['amount'] / 100),
-            bills=bills
+            card_id=int(card_info['acctNo']),
+            amount=float(card_info['acctAmt'] / 100)
         )
 
+    def fetch_bill(self, session: Session) -> List[Bill]:
```

Comment:
抱歉，先前没意识到 `bill` 是个可数名词…
```suggestion
    def fetch_bills(self, session: Session) -> List[Bill]:
```